src/univ/control.cpp

```diff
@@ -69,6 +69,25 @@ bool wxControl::Create(wxWindow *parent,
     return true;
 }
 
+wxControl::~wxControl()
+{
+#if wxUSE_ACCEL
+    wxChar accelChar = GetAccelChar();
+
+    if ( accelChar != wxEMPTY_ACCEL_CHAR )
+    {
+        wxWindow *win = wxGetTopLevelParent(this);
+        if ( win )
```

Comment:
Pretty minor but I think writing it like this:

```suggestion
        if ( wxWindow* win = wxGetTopLevelParent(this) )
```

is more clear (and one line less).

Reply:
Sorry, that's lack of C++ over 20 years. Will change.